### PR TITLE
Choose titles over labels

### PR DESCRIPTION
**Pull request by kate-kazantseva**
## Related Issues

- Links [T-39909](https://transcend.height.app/user/katek/T-39909)

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

File: src/messages.ts [hunk 1]
```diff
@@ -509,11 +509,11 @@ export const completeOptionsMessages = defineMessages(
   'ui.src.completeOptions',
   {
     essentialLabel: {
-      defaultMessage: 'Essential purposes',
+      defaultMessage: 'Essential',
```
Comment (csmccarthy) [inline src/messages.ts:512]: these arent being displayed anywhere anymore, right? why the edits to them?

Comment (kate-kazantseva) [reply src/messages.ts:512]: they are still used as defaults, although now every purpose is supposed to have a title and description. should i remove these and the tests associated with default values?

Comment (kate-kazantseva) [reply src/messages.ts:512]: tests need to be cleaned up for sure, I think some of them use these to compare against

**Review by csmccarthy — APPROVED** [review]
looks great!

**Pull request merged**